packages/mdc-textfield/README.md

```diff
@@ -117,37 +117,6 @@ By default an input's validity is checked via `checkValidity()` on blur, and the
 accordingly. Set the MDCTextField.valid field to set the input's validity explicitly. MDC Text Field
 automatically appends an asterisk to the label text if the required attribute is set.
 
-### Leading and Trailing Icons
```

Comment:
I don't think we should remove this from the top level readme... The top-level readme is all people should need to look at in order to use the component, and they're less likely to look at the subelements' readmes unless they're interested specifically in implementing their own adapters/components.

Reply:
My understanding was that the top-level README has everything that's required to get a basic working text field, and all the "extras" fall into their subelement READMEs. 
I'm not too adamant on having it one way or the other, but if we go with your case then we should also move stuff in helper-text README back to the top-level one.

Reply:
@lynnjepsen suggested that we move as much as possible to the subelement READMEs but mention each subelement in the toplevel README with a "See here for more info" link.

Reply:
Yeah, links would be a good idea at the very least. My main concern is that we're making docs on certain features harder to find otherwise.